platforms/cuda.jl

```diff
@@ -87,6 +87,7 @@ platform(cuda::String) = cuda
 # BinaryBuilder.jl currently does not allow selecting a BuildDependency by compat,
 # so we need the full version for CUDA_SDK_jll (JuliaPackaging/BinaryBuilder.jl#/1212).
 const cuda_full_versions = [
+    v"10.2.89",
```

Comment:
If we want to have the 10.2 version listed here, then I think we need to modify the `supported_platforms()` function below to allow specifying CUDA major versions to target. Build recipes are starting to transition to use `CUDA.supported_platforms()` to get the platform strings, and I don't think all CUDA packages we have will support 10.x.

Reply:
Right, I have added a suggested implementation change in the last push.

Reply:
Bump

Reply:
Bump :-)

Reply:
Bump...